tests/trace_plugin_test.py

```diff
@@ -117,6 +117,7 @@ def setUpClass(self):
     @classmethod
     def tearDownClass(self):
         TraceApiPluginTest.cluster.testFailed = not testSuccessful
+        TraceApiPluginTest.cluster.shutdown()
```

Comment:
Why does this test need to call cluster shutdown explicitly?

Reply:
Because it does not call TestHelper shutdown

Reply:
Oh, only it missed calling the shutdown method.